db/seeds.rb

```diff
@@ -31,6 +31,12 @@ def create_user_for_organisations(
   name: 'Academy for Social Justice Commissioning', service_email: 'empty@example.net'
 )
 
+create_user_for_organisations(
+  [],
+  email: 'loneadmin@gov.uk',
+  confirmed_at: Time.zone.now,
+).update!(is_super_admin: true)
+
```

Comment:
I can't change the rest of the file, but I recommend making the create_user take the `is_super_admin` flag and setting it explicitly both for loneadmin and admin (below)
See https://github.com/alphagov/govwifi-admin/compare/add-empty-superadmin-dashboard#diff-f7a25bc30a409fc77050703fc048ca8c